test/unit/resources/oracledb_session_test.rb

```diff
@@ -15,7 +15,7 @@
 
   it 'run a SQL query' do
     resource = load_resource('oracledb_session', user: 'SYSTEM', password: 'supersecurepass', host: '127.0.0.1', service: 'ORCL.localdomain', port: 1527)
-    query = resource.query('SELECT NAME AS VALUE FROM v$database;')
```

Comment:
this ";" is not needed here, inspec oracle profile contain queries without ";" at the end.

Reply:
If my suggestion is implemented above, this change should not be necessary. Let the resource either add or not add the trailing semicolon when necessary.